pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -1,86 +1,99 @@
+#!bash
```

Comment:
Actually it's a hack so that ShellCheck and vim detect the proper shell dialect. Maybe there is a better way to do it.